addons/im_livechat/static/src/embed/common/chat_window_model_patch.js

```diff
@@ -14,6 +15,7 @@ patch(ChatWindow.prototype, {
         if (this.store.env.services["im_livechat.livechat"].state === SESSION_STATE.PERSISTED) {
             this.hasFeedbackPanel = true;
             this.open({ notifyState: this.thread?.state !== "open" });
+            this.closeFeedback = true;
```

Comment:
`closeFeedback` is useless: the new feature is about showing a leave confirmation dialog, everything happens before feedback screen, and this doesn't change the showing of feedback screen condition from "hasFeedbackPanel" and confirming the closing of chat window.